db/setup.py

```diff
@@ -226,15 +226,24 @@
         "https://sedlabanki.is/",
         "sedlabanki.is",
         "Seðlabankinn",
-        1.0,
+        0.5,
         "scrapers.default",
         "SedlabankinnScraper",
         True,
     ),
+    (
+        "https://bbl.is/",
+        "bbl.is",
+        "Bændablaðið",
+        1.0,
```

Comment:
...and the authority of Bændablaðið to 1.0? :-)

Reply:
Ibid.